### DISCUSSION TITLE
Épocas de entrenamiento y aprendizaje

### DISCUSSION THREAD

**Discussion by LucasMaulen** [JavierBZ/Tarea3-IIC3697#15] [Q&A]
Hola! A las cuantas épocas de entrenamiento deberíamos esperar que la red comience a aprender?

Lleva un rato ejecutándose el algoritmo y me está pasando que no para de obtener 0 de reward total en cada episodio.

**Comment by JavierBZ** [comment]
Hola!

Debería empezar a subir el score desde la quinta o sexta cuando la política e-greedy ya lleva un threshold cercano a 0.1 o 0.1
